z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -394,16 +395,18 @@ Use this playbook to add a new compute node as fixed IP:
 ```sh
 ansible-playbook -i inventory.yaml add-new-compute-node.yaml -e ip=x.x.x.x
 ```
-Use this playbook to add multiple compute nodes as allocated IP, and update bastion info automatically:
+Use this playbook to add multiple compute nodes as allocated IP:
 ```sh
-ansible-playbook -i inventory.yaml add-new-compute-node.yaml -e worker_number=3 -e update_bastion=true
+ansible-playbook -i inventory.yaml add-new-compute-node.yaml -e worker_number=3 
 ```
-Use this playbook to add multiple compute nodes as fixed IP, separate the IP list with commas, and update bastion info automatically:
+Use this playbook to add multiple compute nodes as fixed IP, separate the IP list with commas:
 ```sh
-ansible-playbook -i inventory.yaml add-new-compute-node.yaml -e ip=x.x.x.x,x.x.x.x -e worker_number=2 -e update_bastion=true
+ansible-playbook -i inventory.yaml add-new-compute-node.yaml -e ip=x.x.x.x,x.x.x.x -e worker_number=2
 ```
 **Please notice:**
-> If you use your own bastion server, you can refer [Add-DNS-HAProxy](docs/add-dns-haproxy.md) to update bastion info.
+> We will default treat bastion server and localhost server as a same one.
+> If you use your own bastion server, you can refer [Add-DNS-HAProxy](docs/add-dns-haproxy.md) to update bastion info. 
+> Also modify `use_internal_bastion: false` in inventory.yaml and run Step2 mandatory.
 
```

Comment:
With `use_internal_bastion` configed, we don't need Step2 anymore. Just need give one Note between 01-preparation.yaml and 02-create-cluster-control.yaml . For example:
Note: If you want to use your external or existing DNS and Load Balancer, you Must set `use_internal_bastion: false` and refer [Add-DNS-HAProxy](https://github.com/IBM/z_ansible_collections_samples/blob/main/z_infra_provisioning/cloud_infra_center/ocp_upi/docs/add-dns-haproxy.md) to update DNS and Load Balancer records manually.